src/ui/api-models/Clause.ts

```diff
@@ -10,7 +10,7 @@ import * as t from "io-ts";
 const ClauseR = t.interface({
   title: t.string,
 
-  required: t.boolean,
+  required: t.boolean
```

Comment:
I'm unsure if we want to commit generated files. cc @lucacavallaro what about it?

Reply:
for me it is better not to commit them.

Reply:
deal, put them into some `generated` folder then add them to .gitingore

Reply:
in this way we can delete the ui folder?

Reply:
Yes if it just contains generated files. 

Consider to generate the types for queue message (from a different yaml) as well. In other words, since other processes (external to this repo) may read the queue messages, better to threat them as API types.

Reply:
ok I move everything to a `generated` folder. For now, the types for queue messages are used only by this repository. However as soon as possible we will create a dedicated yaml.